frontend/src/components/main-page/auth-tokens-page/auth-tokens-page.tsx

```diff
@@ -0,0 +1,10 @@
+import { Box } from '@mui/material';
+import React from 'react';
+
+export const AuthTokensPage: React.FC = () => {
+  return (
+    <Box sx={{ height: '100%', overflowY: 'auto' }}>
+      <h3>Auth Tokens</h3>
```

Comment:
API tokens?

Reply:
Good point, `AuthTokens` is definitely not a good term 😄 I think I will call it `API Keys` to match the name used on the server. (Will do this renaming once I actually start working on this page.)